frontend/src/components/common/Terminal.tsx

```diff
@@ -197,8 +197,30 @@ export default function Terminal(props: TerminalProps) {
       shellConnectFailed(xtermc);
       return;
     }
-
-    xterm.write(text);
+    // replace \n with \r only if there is \r before \n
+    // give me the regular expression for the above
+    text.replace(/\r\n/g, '\r').replace(/\n/g, '\r');
+    if (props.isAttach) {
+      // in case of attach if we didn't recieve any data from the process we should try to reconnect
+      if (!text) {
+        text = "If you don't see a command prompt, try pressing enter.";
+      }
```

Comment:
The terminal is not responding to any input, so we should remove this message as it's confusing.

Reply:
Can we check if, once the attach is successful, we just print a message saying "Any new output for this container's process should be shown below."